build.xml

```diff
@@ -1,35 +1,101 @@
-<project name="Documentation" default="all.doc" >
-   <property name="pdf.name" value="bean-validation-specification.pdf"/>
-   <property name="spec.styles.dir" value="${basedir}/styles"/>
-   <property name="document.basedir" value="${basedir}"/>
-
-
-   <import file="${document.basedir}/docbook-support/support.xml"/>
-
-   <target name="all.doc" depends="clean">
-      <antcall target="lang.all">
-         <param name="lang" value="en"/>
-         <param name="basedir" value="${document.basedir}"/>
-         <param name="styles.dir" value="${spec.styles.dir}"/>
-      </antcall>
-   </target>
-
-   <target name="clean">
-      <delete dir="${document.basedir}/build" />
-   </target>
-
-   <target name="createTckAuditFile">
```

Comment:
Sure, I can restore them, but I don't understand why you'd delete them in the first place. It's an essential part of the overall mechanism. At least as a "reminder" they should remain in place, the task is not done without this piece working.